### ISSUE TITLE
The result of completing the 80 epoch is very poor and cannot be duplicated, and ask for your help.

### ISSUE THREAD

**Issue by QiyuLuo** [ifzhang/ByteTrack#89]
I use 2 rtx titan 24G, batch_size 8 to train,
Namespace(batch_size=8, ckpt='YOLOX_outputs/yolox_x_mix_det/last_epoch_ckpt.pth.tar', devices=2, dist_backend='nccl', dist_url=None, exp_file='exps/example/mot/yolox_x_mix_det.py', experiment_name='yolox_x_mix_det', fp16=True, local_rank=0, machine_rank=0, name=None, num_machines=1, occupy=True, opts=[], resume=True, start_epoch=18)
╒══════════════════╤═══════════════════╕
│ keys             │ values            │
╞══════════════════╪═══════════════════╡
│ seed             │ None              │
├──────────────────┼───────────────────┤
│ output_dir       │ './YOLOX_outputs' │
├──────────────────┼───────────────────┤
│ print_interval   │ 20                │
├──────────────────┼───────────────────┤
│ eval_interval    │ 5                 │
├──────────────────┼───────────────────┤
│ num_classes      │ 1                 │
├──────────────────┼───────────────────┤
│ depth            │ 1.33              │
├──────────────────┼───────────────────┤
│ width            │ 1.25              │
├──────────────────┼───────────────────┤
│ data_num_workers │ 4                 │
├──────────────────┼───────────────────┤
│ input_size       │ (800, 1440)       │
├──────────────────┼───────────────────┤
│ random_size      │ (18, 32)          │
├──────────────────┼───────────────────┤
│ train_ann        │ 'train.json'      │
├──────────────────┼───────────────────┤
│ val_ann          │ 'val_half.json'   │
├──────────────────┼───────────────────┤
│ degrees          │ 10.0              │
├──────────────────┼───────────────────┤
│ translate        │ 0.1               │
├──────────────────┼───────────────────┤
│ scale            │ (0.1, 2)          │
├──────────────────┼───────────────────┤
│ mscale           │ (0.8, 1.6)        │
├──────────────────┼───────────────────┤
│ shear            │ 2.0               │
├──────────────────┼───────────────────┤
│ perspective      │ 0.0               │
├──────────────────┼───────────────────┤
│ enable_mixup     │ True              │
├──────────────────┼───────────────────┤
│ warmup_epochs    │ 1                 │
├──────────────────┼───────────────────┤
│ max_epoch        │ 80                │
├──────────────────┼───────────────────┤
│ warmup_lr        │ 0                 │
├──────────────────┼───────────────────┤
│ basic_lr_per_img │ 1.5625e-05        │
├──────────────────┼───────────────────┤
│ scheduler        │ 'yoloxwarmcos'    │
├──────────────────┼───────────────────┤
│ no_aug_epochs    │ 10                │
├──────────────────┼───────────────────┤
│ min_lr_ratio     │ 0.05              │
├──────────────────┼───────────────────┤
│ ema              │ True              │
├──────────────────┼───────────────────┤
│ weight_decay     │ 0.0005            │
├──────────────────┼───────────────────┤
│ momentum         │ 0.9               │
├──────────────────┼───────────────────┤
│ exp_name         │ 'yolox_x_mix_det' │
├──────────────────┼───────────────────┤
│ test_size        │ (800, 1440)       │
├──────────────────┼───────────────────┤
│ test_conf        │ 0.001             │
├──────────────────┼───────────────────┤
│ nmsthre          │ 0.7               │
├──────────────────┼───────────────────┤
│ data_name        │ 'mix_det'         │
├──────────────────┼───────────────────┤
│ name             │ ''                │
├──────────────────┼───────────────────┤
│ val_name         │ 'train'           │
╘══════════════════╧═══════════════════╛
after finished 80 epochs, 
![image](https://user-images.githubusercontent.com/34224700/143398404-da001852-0230-40a3-8adb-b0d702358def.png)
i get bad results compared with the results in the training set in the paper, so I would like to ask for your help on what is wrong.
**paper**:
![image](https://user-images.githubusercontent.com/34224700/143398886-e63853cd-0c9f-4a67-b979-cd0365c6698a.png)
**my results**:
![QQ截图20211125153558](https://user-images.githubusercontent.com/34224700/143399213-4ef81ba9-41e6-475b-97bb-a2ee12ac89bc.png)

**Comment by yanghaibin-cool** [comment]
I compared the weights given by the author and found that the detector was not well trained, so I don't know how to adjust the parameters

**Comment by QiyuLuo** [comment]
> I compared the weights given by the author and found that the detector was not well trained, so I don't know how to adjust the parameters
Strangely, when I trained the ablation experiment, the results were similar to those given by the authors in half of the validation sets, but there was a big difference in training the entire data set.

**Comment by yanghaibin-cool** [comment]
Me too. The ablation experiment results were similar. When I used the weights I trained on the complete data set to track, THE COCO index of the verification set was not as good as the weight given by the author, which also led to the failure of the MOT index to reach the value stated by the author. But it bothers me not to know what the problem is.

**Comment by ifzhang** [comment]
Thanks for your experiments! The mix dataset code only mix half mot17 training set. I have fixed it now.

**Comment by QiyuLuo** [comment]
> l
The author replies that the script for mixing datasets is half mot17. 
@ifzhang thank your for your help,  can i just train the MOT17 to do fine tuning with this model.

**Comment by ifzhang** [comment]
You'd better train again using all the datasets. The performance on the test set is better.

**Comment by QiyuLuo** [comment]
> You'd better train again using all the datasets. The performance on the test set is better.

thank your for your advice. I don't have so many gpus. It takes too long to train once. I will try it first, If the effect is still much worse, I will retrain it.
